tests/providers/mysql/operators/test_s3_to_mysql.py

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest.mock import patch
+
+from airflow import configuration, models
+from airflow.providers.mysql.operators.s3_to_mysql import S3ToMySqlTransfer
+from airflow.utils import db
+
+
+class TestS3ToMySqlTransfer(unittest.TestCase):
+
+    def setUp(self):
+        configuration.conf.load_test_config()
+
+        db.merge_conn(
+            models.Connection(
+                conn_id='s3_test',
+                conn_type='s3',
+                schema='test',
+                extra='{"aws_access_key_id": "aws_access_key_id", "aws_secret_access_key":'
+                      ' "aws_secret_access_key"}'
+            )
+        )
+        db.merge_conn(
+            models.Connection(
+                conn_id='mysql_test',
+                conn_type='mysql',
+                host='some.host.com',
+                schema='test_db',
+                login='user',
+                password='password'
+            )
+        )
```

Comment:
Can we remove these connections in tearDown please?

Reply:
Good point.